src/test/java/seedu/address/logic/commands/AddHeightCommandTest.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.FitNus;
+import seedu.address.model.ReadOnlyFitNus;
+import seedu.address.model.person.Height;
+import seedu.address.testutil.HeightBuilder;
+
+public class AddHeightCommandTest {
```

Comment:
Any other possible test cases? Seems like you are only considering null as an input. Maybe think about the equivalence partitions